index.js

```diff
@@ -1,42 +1,58 @@
-/* eslint-disable no-unused-vars */
+/* eslint-disable import/named */
+import BookClass from './bookClass.js';
 /* eslint-disable no-use-before-define */
-const form = document.getElementById('form');
-const bookList = document.getElementById('bookList');
-const btn = document.getElementById('btn');
-let books = JSON.parse(localStorage.getItem('Atif')) || [];
+/* eslint-disable no-unused-vars */
+class Books {
+  constructor() {
+    this.bookList = document.getElementById('bookList');
+    this.bookTitle = document.getElementById('titleInput');
+    this.bookAuthor = document.getElementById('authorInput');
+    this.form = document.getElementById('form');
+    this.booksArray = [];
+    if (localStorage.getItem('Atif')) {
+      this.booksArray = JSON.parse(localStorage.getItem('Atif'));
+      this.displayBooks();
+    }
+  }
```

Comment:
Here it's always good to use a single task class of function class Books is meant to manage list of books retrieving data from the form can be outside the the class so the constructor here should just initialize the list of books and it can be done in one line 
`this.booksArray = JSON.parse(localStorage.getItem('Atif')) || [] ;`

Reply:
I have made this change as well.